core/src/main/java/org/apache/iceberg/hadoop/HadoopCatalog.java

```diff
@@ -93,12 +94,14 @@ public class HadoopCatalog extends BaseMetastoreCatalog implements Closeable, Su
   private FileIO fileIO;
   private LockManager lockManager;
   private boolean suppressPermissionError = false;
+  private Map<String, String> catalogProperties = Collections.emptyMap();
```

Comment:
Nit / non-blocking : Does this _need_ to be initialized? I'm assuming this is to suppress potential not-initialized errors?